astroquery/ipac/nexsci/nasa_exoplanet_archive/core.py

```diff
@@ -387,18 +389,36 @@ def query_aliases(self, object_name, *, cache=None):
         response : list
             A list of aliases found for the object name. The default name will be listed first.
         """
-        return list(
-            self.query_criteria(
-                "aliastable", objname=object_name.strip(), cache=cache, format="csv"
-            )["aliasdis"]
-        )
+        url = requests.get("https://exoplanetarchive.ipac.caltech.edu/cgi-bin/Lookup/nph-aliaslookup.py?objname="+object_name)
```

Comment:
I think this messes with the assumption in the mock test that expects an URL derived from the config, maybe these lines need to be revised when testing the alias resolution:

```
def mock_get(self, method, url, *args, **kwargs):  # pragma: nocover
    assert url == conf.url_api
 ```